external-crates/move/crates/move-compiler/tests/move_2024/parsing/clever_errors_raw_abort.move

```diff
@@ -0,0 +1,39 @@
+#[allow(dead_code)]
```

Comment:
I'm about confused about these tests. Should these all be passing?

Reply:
ah yea, they should be. I'll add the `allow(unused_trailin_semi)` here.

Reply:
All fixed now.